package.json

```
@@ -65,6 +65,7 @@
     "ci:test": "npm run test",
     "clean": "rm -rf typechain-types cache bin dist artifacts contracts-exposed",
     "postclean": "hardhat clean",
+    "collector:deploy": "hardhat deploy-collector",
```

probably better the other way.
```suggestion
    "deploy:collector": "hardhat deploy-collector",
```

---

Not sure about it. I was thinking about grouping all the collector related scripts, so we would have something like: `collector:deploy`, `collector:withdraw`, `collector:change-partners`. WDYT?